src/sidenav.ts

```diff
@@ -202,7 +205,7 @@ export class Sidenav extends Component<SidenavOptions> implements Openable {
     const trigger = e.target.closest('.sidenav-trigger');
     if (e.target && trigger) {
       const sidenavId = Utils.getIdFromTrigger(trigger);
-      const sidenavInstance = (document.getElementById(sidenavId) as any).M_Sidenav;
+      const sidenavInstance = document.getElementById(sidenavId)['M_Sidenav'];
```

Comment:
I think we might need to revert this, since it introduces backwards compatibility issues with applications using the previous object orientated approach, propose to just set it to warnings until we can declare the correct type definition